phpunit.xml.dist

```diff
@@ -3,7 +3,7 @@
 <!-- http://phpunit.de/manual/current/en/appendixes.configuration.html -->
 <phpunit
         colors="true"
-        bootstrap="vendor/autoload.php">
+        bootstrap="phpunit.php">
```

Comment:
I'll move it, but I used the name for this file because Laravel does it too, I thought it would be a good idea to follow the pattern. What do you think?